languages/nonBuildable.groovy

```diff
@@ -0,0 +1,77 @@
+@groovy.transform.BaseScript com.ibm.dbb.groovy.ScriptLoader baseScript
+import com.ibm.dbb.repository.*
+import com.ibm.dbb.dependency.*
+import com.ibm.dbb.build.*
+import com.ibm.dbb.build.report.records.*
+import com.ibm.dbb.build.report.*
+import groovy.transform.*
+
+/***
+ * Language script, which just copies files to the defined target dataset 
+ * and reports the file as a build output file in the build report
+ */
+
+// define script properties
+@Field BuildProperties props = BuildProperties.getInstance()
+@Field def buildUtils= loadScript(new File("${props.zAppBuildDir}/utilities/BuildUtilities.groovy"))
+// Set to keep information about which datasets where already checked/created
+@Field HashSet<String> verifiedBuildDatasets = new HashSet<String>()
+
+@Field RepositoryClient repositoryClient
+
+println("** Building files mapped to ${this.class.getName()}.groovy script")
+
+// verify required build properties
+buildUtils.assertBuildProperties(props.cobol_requiredBuildProperties)
```

Comment:
Should be nonbuildable_requiredBuildProperties ?

Reply:
Good catch. This should be fixed.